controllers/page/PageHandler.inc.php

```diff
@@ -164,7 +164,7 @@ private function setupHeader($args, $request) {
 			$dispatcher = $request->getDispatcher();
 			$contextsNameAndUrl = array();
 			while ($workingContext = $workingContexts->next()) {
-				$contextUrl = $dispatcher->url($request, ROUTE_PAGE, $workingContext->getPath());
+				$contextUrl = $dispatcher->url($request, ROUTE_PAGE, $workingContext->getPath(), 'dashboard');
```

Comment:
I thought there was a drop-down at the top of the page -- was that removed? Perhaps that's not a bad idea, though we'll probably get some feedback about it.